src/j1cli.js

```diff
@@ -447,6 +456,15 @@ async function mutateQuestions(j1Client, questions, operation) {
       )}.`
     );
   }
+  if (newFile) {
+    var stringify = JSON.stringify(questions, null, 4);
+    fs.writeFile("modified_questions.json", stringify, function(err) {
```

Comment:
This is an async call that is out of band. We should make it in-band instead. You can accomplish this by using `util.promisify` and awaiting on `fs.writeFileAsync`.

Reply:
Eh, its a CLI.  We can do it sync. Lets change it to `writeFileSync`

Reply:
Should we stick with your idea or Rausch's here

Reply:
Use Rausch's. `writeFileSync` is perfect for this use case IMO.

Reply:
@mnguo98 and I discussed this a bit in person. Because the learning would be valuable, I'd go @austinkelleher's suggestion (which @mnguo98 actually already pushed up). :smile: